File: A4.Empower-Git/A4.Empower/A4.BAL/Leave/LeaveCalenderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace A4.BAL
{
    public class LeaveCalenderModel
    {
        public string Title { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public bool AllDay { get; set; }
        public string BackgroundColor { get; set; }
        public string BorderColor { get; set; }
        public string Tooltip { get; set; }
        public string Status { get; set; }
    }
}

File: ClientApp/src/app/models/leave/leave-calender.model.ts
export class LeaveCalender {

    constructor(title?: string, start?: Date, end?: Date, allDay?: boolean, backgroundColor?: string, borderColor?: string, tooltip?: string, status?: string) {

        this.title = title;
        this.start = start;
        this.end = end;
        this.allDay = allDay;
        this.backgroundColor = backgroundColor;
        this.borderColor = borderColor;
        this.tooltip = tooltip;
        this.status = status;

    }

    public title: string;
    public start: Date;
    public end: Date;
    public allDay: boolean;
    public backgroundColor: string;
    public borderColor: string;
    public tooltip: string;
    public status: string;
}